docs/source/quicktour.mdx

```diff
@@ -211,220 +211,93 @@ and get tensors back. You can specify all of that to the tokenizer:
+The model outputs also behave like a tuple or a dictionary (e.g., you can index with an integer, a slice or a string) in which
+case the attributes that are `None` are ignored.
 </Tip>
 
-Once your model is fine-tuned, you can save it with its tokenizer in the following way:
+## Save your model
+
+Once your model is fine-tuned, you can save it with its tokenizer using `save_pretrained`:
 
-```py
->>> pt_save_directory = './pt_save_pretrained'
+```python
+>>> pt_save_directory = "./pt_save_pretrained"
 >>> tokenizer.save_pretrained(pt_save_directory)
 >>> pt_model.save_pretrained(pt_save_directory)
 ===PT-TF-SPLIT===
->>> tf_save_directory = './tf_save_pretrained'
+>>> tf_save_directory = "./tf_save_pretrained"
 >>> tokenizer.save_pretrained(tf_save_directory)
 >>> tf_model.save_pretrained(tf_save_directory)
 ```
 
-You can then load this model back using the [`AutoModel.from_pretrained`] method by passing the
-directory name instead of the model name. One cool feature of 🤗 Transformers is that you can easily switch between
-PyTorch and TensorFlow: any model saved as before can be loaded back either in PyTorch or TensorFlow.
+When you are ready to use the model again, reload it with `from_pretrained`:
 
-
-If you would like to load your saved model in the other framework, first make sure it is installed:
-
-```bash
-pip install torch
+```python
+>>> pt_model = AutoModelForSequenceClassification.from_pretrained("./pt_save_pretrained")
 ===PT-TF-SPLIT===
-pip install tensorflow
+>>> tf_model = TFAutoModelForSequenceClassification.from_pretrained("./tf_save_pretrained")
 ```
 
-Then, use the corresponding Auto class to load it like this:
+One particularly cool 🤗 Transformers feature is the ability to save a model and reload it as either a PyTorch or TensorFlow model. The `from_pt` or `from_tf` parameter can convert the model from one framework to the other:
 
-```py
+```python
 >>> from transformers import AutoModel
 >>> tokenizer = AutoTokenizer.from_pretrained(tf_save_directory)
->>> pt_model = AutoModel.from_pretrained(tf_save_directory, from_tf=True)
+>>> pt_model = AutoModelForSequenceClassification.from_pretrained(tf_save_directory, from_tf=True)
 ===PT-TF-SPLIT===
 >>> from transformers import TFAutoModel
 >>> tokenizer = AutoTokenizer.from_pretrained(pt_save_directory)
->>> tf_model = TFAutoModel.from_pretrained(pt_save_directory, from_pt=True)
-```
-
-Lastly, you can also ask the model to return all hidden states and all attention weights if you need them:
-
-
-```py
->>> pt_outputs = pt_model(**pt_batch, output_hidden_states=True, output_attentions=True)
->>> all_hidden_states  = pt_outputs.hidden_states 
->>> all_attentions = pt_outputs.attentions
-===PT-TF-SPLIT===
->>> tf_outputs = tf_model(tf_batch, output_hidden_states=True, output_attentions=True)
->>> all_hidden_states =  tf_outputs.hidden_states
->>> all_attentions = tf_outputs.attentions
-```
-
-### Accessing the code
```

Comment:
I don't think we should remove this from here if we are not adding it somewhere else.

Reply:
For sure, we will add this section in the guides as an alternative way to load a model.